src/pages/myNotes/notes.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+
+import { Container, NotesList, StyledListComponent } from "./styled";
```

Comment:
move styled import at the end of this import list.

Reply:
Moved